docs/source/projects.rst

```diff
@@ -1,6 +1,13 @@
 Projects
 ========
 
+**Edge**
+^^^^^^^^^^^^^^^^^^
```

Comment:
What about, 
edge-sandbox ?

Reply:
i'll add that. i previously thought it was for demos only

Reply:
done @talwinder50